docs/core/releases.md

```diff
@@ -0,0 +1,46 @@
+# Releases
+
+## Versioning
+
+etclabscore/core-geth uses [Semantic Versioning](https://semver.org). The API definition that would demand increments to the major version is basically nil;
+it can be expected that a major version bump would be accompanied by an entirely new repository and name.
+
+Tagged versions use the suffix `-stable` and untagged versions (ie everything else) uses the `-unstable` suffix.
+
+You can find some historical discussions on versioning at the following links.
+- https://github.com/etclabscore/core-geth/pull/29#issuecomment-588977383  
+- https://github.com/etclabscore/multi-geth-fork/issues/153
+- https://github.com/etclabscore/core-geth/pull/30#issuecomment-591979271
+- https://github.com/etclabscore/core-geth/issues/83
+
+
+## Developers: How to make a release
+
+- [ ] Decide what the new version should be. In this example, __`v1.11.16[-stable]`__ will be used. 
+- [ ] `git checkout master`
+- [ ] `make lint` and `make test` are passing on master. :white_check_mark:
+  > This is important because the artifacts to be included with the release will be generated
+  by the CI workflows. If linting or tests fail, the workflows will be interrupted
+  and artifacts will not be generated.
+- [ ] `git checkout release/v1.11.16`
+- [ ] Edit `params/version.go` making the necessary changes to version information. (To `-stable` version.) _Gotcha:_ make sure this passes linting, too.
+- [ ] `git commit -S -s -m "bump version from v1.11.15-unstable to v1.11.16-stable"`
+- [ ] `git tag -S -a v1.11.16`
+- [ ] `git push etclabscore v1.11.16`
+  > Push the tag to the remote. I like to do it this way because it triggers the tagged version on CI before the branch/PR version,
+  expediting artifact delivery.
+- [ ] Edit `params/version.go` making the necessary changes to version information. (To `-unstable` version.)
+- [ ] `git commit -S -s -m "bump version from v1.11.15-unstable to v1.11.17-unstable"`
```

Comment:
Based on previous comment we are at `v.1.11.16-stable`. If true then the cmd might be:

`git commit -S -s -m "bump version from v1.11.16-stable to v1.11.17-unstable"`

Reply:
https://github.com/etclabscore/core-geth/pull/217/commits/42f7ea6b3d7069c848c58b08c32cdb3602e4ba69